source/config/Config.cpp

```diff
@@ -354,10 +354,6 @@ bool PlainConfig::Validate() const
         LOGM_ERROR(Config::TAG, "*** %s: Certificate is missing ***", DeviceClient::DC_FATAL_ERROR);
         return false;
     }
-    else if (!FileUtils::IsValidFilePath(cert->c_str()))
```

Comment:
I am a little bit confused on this, don't we still need to have this if we pass in params via CLI?